presto-product-tests/bin/run_on_docker.sh

```diff
@@ -90,12 +106,24 @@ docker-compose -f "${DOCKER_COMPOSE_LOCATION}" pull
 
 # start hadoop container
 docker-compose -f "${DOCKER_COMPOSE_LOCATION}" up -d hadoop-master
-docker-compose -f "${DOCKER_COMPOSE_LOCATION}" logs --no-color > "${DOCKER_LOGS_LOCATION}" 2>&1 &
+
+# start docker logs to be able to check what is going on spawning hadoop-master container
+start_docker_logs
```

Comment:
maybe start logs at the begin and stop at the end instead of startiing/stoping multiple times